### PR TITLE
redirect after login & login button after logout

### PR DESCRIPTION
**Pull request by karol7531**

### CODE REVIEW

File: Main/ClientApp/src/Authorization/Logout.js [hunk 1]
```diff
@@ -59,7 +61,19 @@ export class Logout extends Component {
       return <div></div>;
     }
     if (!!message) {
-      return <PanelWideMessage text={message} />;
+      return (
```
Comment (arkadR) [inline Main/ClientApp/src/Authorization/Logout.js:64]: Personally. I'd rather just redirect the user to the home page without any message or button, he can log back in from there. But this is also fine, you be the judge

File: Main/ClientApp/src/Components/WelcomeScreen.tsx [hunk 2]
```diff
@@ -1,11 +1,31 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import PanelWideMessage from "./Common/PanelWideMessage";
 import Layout from "./Layout";
+import authService from "../Authorization/AuthorizeService";
+import { Redirect } from "react-router-dom";
 
 export default function WelcomeScreen() {
```
Comment (arkadR) [inline Main/ClientApp/src/Components/WelcomeScreen.tsx:7]: BTW, I had an idea to display some screenshots/tutorials on the home page with a button "Get started" that would either redirect to collections or login page. 
This is also fine, but if I have time, I will do a nicer home page

Comment (karol7531) [reply Main/ClientApp/src/Components/WelcomeScreen.tsx:7]: that home page would be great, but for now this is better than "Log in to get started" caption

File: Main/ClientApp/src/Authorization/Logout.js (hunk 1 above)
Comment (karol7531) [reply Main/ClientApp/src/Authorization/Logout.js:64]: redirect is better option, i'll change that

**Pull request merged**